src/components/groups/PostList.tsx

```
@@ -96,85 +132,85 @@ export function PostList({ communityId, showOnlyApproved = false, pendingOnly =
+      }).filter((post): post is NostrEvent & {
+        approval: { id: string; pubkey: string; created_at: number; kind: number }
       } => post !== null);
-      
+
       // Debug logging
       console.log("Filtered approved posts:", {
         totalApprovedPosts: approvedPosts.length
       });
-      
+
       return approvedPosts;
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)